server/protocols/jmap-integration-testing/pom.xml

```diff
@@ -34,6 +34,7 @@
 
     <modules>
         <module>jmap-integration-testing-common</module>
+        <module>memory-jmap-integration-testing</module>
```

Comment:
you should not do that, anyway profiles are an old story now, a rebase should do the job

Reply:
Sure, that is just a mistake...